src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup/Properties/AssemblyInfo.cs

```diff
@@ -41,5 +41,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("4.1.3")]
-[assembly: AssemblyFileVersion("4.1.3")]
+[assembly: AssemblyVersion("4.2.0")]
```

Comment:
I think that the other recoveryservices.backup project need this change as well.

Reply:
done